notebooks/search/04-multilingual.ipynb

```diff
@@ -37,7 +37,7 @@
     "\n",
     "- Python 3.6 or later\n",
     "- An Elastic deployment with minimum **4GB machine learning node**\n",
-    "   - We'll be using [Elastic Cloud](https://www.elastic.co/guide/en/cloud/current/ec-getting-started.html) for this example (available with a [free trial](https://cloud.elastic.co/registration?elektra=en-ess-sign-up-page))\n",
+    "   - We'll be using [Elastic Cloud](https://www.elastic.co/guide/en/cloud/current/ec-getting-started.html) for this example (available with a [free trial](https://cloud.elastic.co/registration?utm_source=github&utm_content=elasticsearch-labs-notebook?elektra=en-ess-sign-up-page))\n",
```

Comment:
remove `?elektra=en-ess-sign-up-page`

Reply:
added in 5725f74106dc37b600e6ba7f4f4433ad20e7bb7c